ebsco/edsapi-koha-plugin
# Various Adjustments to Setup and Cache

Removes session cache of admin settings causing users to not see changes reflected immediately.

Updates setup information.

Removes jQuery script setup information that's no longer necessary and was causing cart duplications when installed.

Removes remaining references to rawgit, replacing them with jsdelivr.

To Test:

Install the 19.05010.kpz file in your test Koha instance.

Verify install is successful and setup instructions have been updated to reflect reference to connect.ebsco.com.

Open the configuration screen and verify that information on jquery string has been removed from the default search instructions.

After configuring site, set your default search to "EDS", and save your settings.  Verify in your main interface that Discovery is the default that appears (this may take a second to load).

Go back to configuration, set your default search to "Koha", and save your settings.  Verify in your main interface that Catalog is the default search selected.
